cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -72,6 +72,8 @@ class MappingStep(CCIDictModel):
     static: Dict[str, str] = {}
     filters: List[str] = []
     action: str = "insert"
+    api: DataApi = DataApi.SMART
+    batch_size: int = 200
     oid_as_pk: bool = False  # this one should be discussed and probably deprecated
     record_type: Optional[str] = None  # should be discussed and probably deprecated
     bulk_mode: Optional[
```

Comment:
Should we add a validator to make sure the batch size can't be set too high (i.e. with different limits based on which api is specified)?

Reply:
Great idea. Adding.